src/site/_includes/partials/post.njk

```diff
@@ -56,13 +56,11 @@
 
     {{ content | safe }}
 
-    {# Hide tags for now. #} {#
     <div class="w-chips">
-      <a class="w-chip" href="#">Artificial Intellegence</a>
-      <a class="w-chip" href="#">User Experience</a>
-      <a class="w-chip" href="#">Learning</a>
+      {% for tag in tags | findTags %}
+        <a class="w-chip" href="{{ tag.href }}">{{ tag.title }}</a>
```

Comment:
Won't the href always be `/tags/{{tag}}` ? Just wondering if we could simplify to that.

Reply:
Yeah, this might remove the extra work in modifying the tags in their config file.

Reply:
Tis used in `_includes/partials/paged.njk` and can't/shouldn't be modified there. The partial uses data generated by `_utils/add-pagination.js`, which is used by tags and blog posts.